### DISCUSSION TITLE
Beta Projects - Navigation from Project back to Organisation Unintuitive

### DISCUSSION THREAD

**Discussion by dataGriff** [community/community#17434] [Projects and Issues] [answered]
When you are in the new project board beta it is not very intuitive of how you can get back to the organisation level assets such as repos and the organisation itself without going back through all the main menus etc.

The command pallete offers just project options and not navigation routes so this does not work as a quick journey method.

Its surprisingly frustrating and feels like it takes off some of the shine off the inner project beta abilities that are now available.

**Comment by dataGriff** [comment]
apologies this is a replica #10448
this should be removed

**Comment by 2percentsilk** [comment, marked as answer]
We've added a quick fix to help you [navigate](https://github.blog/changelog/2022-06-02-the-new-github-issues-june-2nd-update/#%F0%9F%A7%AD-improved-navigation-via-project-header)! It doesn't support navigating to repos but it is a first step in helping you get back to the organization level view.

Let us know if that helps!
